src/test/java/org/ibp/api/java/impl/middleware/ontology/validator/VariableValidatorTest.java

```diff
@@ -303,19 +304,22 @@ public void testWithSpecialCharacterAnsStartWithDigitVariableName() throws Middl
 	}
 
 	/**
-	 * Test for Alias contain special character and start with digit
+	 * Test for variables with the same Name
 	 */
 	@Test
-	public void testWithSpecialCharacterAnsStartWithDigitVariableAlias() throws MiddlewareException {
+	public void testForVariablesWithTheSameName() throws MiddlewareException {
```

Comment:
I think we should add a case for name and another one for alias

Reply:
done.